src/rpc/misc.cpp

```diff
@@ -467,11 +467,11 @@ UniValue verifymessage(const JSONRPCRequest& request)
             "\nUnlock the wallet for 30 seconds\n"
             + HelpExampleCli("walletpassphrase", "\"mypassphrase\" 30") +
             "\nCreate the signature\n"
-            + HelpExampleCli("signmessage", "\"XwnLY9Tf7Zsef8gMGL2fhWA9ZmMjt4KPwg\" \"my message\"") +
+            + HelpExampleCli("signmessage", "\"XwnLY9Tf7Zsef8gMGL2fhWA9ZmMjt4KPwG\" \"my message\"") +
```

Comment:
there are more examples that should be fixed in a similar way

Reply:
How did the invalid one make it into the comment anyway?

Reply:
The invalid ones are intentional so that people don't send Dash to it by accident. This is part of https://github.com/bitcoin/bitcoin/pull/9865 (but with Dash addresses)